apps/openmw/mwmechanics/aiwander.cpp

```diff
@@ -227,8 +239,20 @@ namespace MWMechanics
         }
 
         // Actor becomes stationary - see above URL's for previous research
-        if(mAllowedNodes.empty())
+        // If not an NPC and no pathgrid is available, randomly idle or wander around near spawn point
+        if(mAllowedNodes.empty() && !actor.getBase()->mClass->isNpc() && !storage.mIsWanderingManually) {
```

Comment:
From a quick test in the original engine, it seems that NPCs are also affected by this behaviour.

(FWIW, there would have been a shorter way to write the statement: actor.getClass().isNpc() )